constants.py

```diff
@@ -196,6 +205,12 @@ class Permission:
 ITEM_NAME = "name"
 ITEM_DESC = "desc"
 ITEM_BULK = "bulk"
-ITEM_TYPE = "type"
+ITEM_TYPE = "type" # Special types used by the frontend are: "equipment", "consumable", "container", and "weapon"

```

Comment:
The server shouldn't care what the frontend does, right?

Reply:
yes - just noting them there in a comment for now. We could probably make that an enum in the future if we thought it was necessary, but I don't think it is